server/aws-lsp-codewhisperer/src/language-server/agenticChat/qAgenticChatServer.ts

```diff
@@ -47,7 +58,7 @@ export const QAgenticChatServer =
                             ],
                         },
                         mcpServers: enabledMCP(params),
-                        modelSelection: true,
+                        modelSelection: modelSelectionEnabled,
```

Comment:
can this logic follow the same pattern as enabledMcp defined in the utils class to determine modelSelectionEnabled value on L41?